seacrowd/sea_datasets/bloom_captioning/bloom_captioning.py

```diff
@@ -0,0 +1,249 @@
+}
+"""
+
+logger = datasets.logging.get_logger(__name__)
+
+# this config is created for SEACrowd Dataloader
+_LANG_CONFIG = {
+    "abc": "Ambala Ayta",
+    "ahk": "Akha",
+    "bfn": "Bunak",
+    "bjn": "Banjar",
+    "bkx": "Baikeno",
+    "brb": "Brao",
+    "brv": "Western Bru",
+    "bya": "Batak",
+    "bzi": "Bisu",
+    "ceb": "Cebuano",
+    "cgc": "Kagayanen",
+    "cmo": "Central Mnong",
+    "ddg": "Fataluku",
+    "dmg": "Upper Kinabatangan",
+    "dnw": "Western Dani",
+    "dtp": "Kadazan Dusun",
+    "dtr": "Lotud",
+    "enc": "En",
+    "fil": "Filipino",
+    "gal": "Galolen",
+    "hil": "Hiligaynon",
+    "hre": "Hre",
+    "hro": "Haroi",
+    "idt": "Idaté",
+    "ilo": "Ilocano",
+    "ind": "Indonesian",
+    "jra": "Jarai",
+    "kak": "Kalanguya",
+    "khb": "Lü",
+    "khm": "Khmer",
+    "kqr": "Kimaragang",
+    "krr": "Krung",
+    "ksw": "S’gaw Karen",
+    "lhu": "Lahu",
+    "llg": "Lole",
+    "lsi": "Lacid",
+    "lwl": "Eastern Lawa",
+    "mdr": "Mandar",
+    "mgm": "Mambae",
+    "mhx": "Lhao Vo",
+    "mkz": "Makasae",
+    "mnw": "Mon",
+    "mqj": "Mamasa",
+    "mry": "Mandaya",
+    "msb": "Masbatenyo",
+    "mya": "Burmese",
+    "nod": "Northern Thai",
+    "nst": "Tangshang Naga",
+    "nxa": "Nauete",
+    "nxl": "South Nuaulu",
+    "pag": "Pangasinan",
+    "pce": "Ruching Palaung",
+    "pdu": "Kayan",
+    "pea": "Peranakan Indonesian",
+    "pmf": "Pamona",
+    "sea": "Semai",
+    "sgd": "Surigaonon",
+    "shn": "Shan",
+    "sml": "Central Sama",
+    "snl": "Sangil",
+    "tdt": "Tetun Dili",
+    "tet": "Tetun",
+    "tha": "Thai",
+    "tkd": "Tukudede",
+    "tnt": "Tontemboan",
+    "tom": "Tombulu",
+    "tpu": "Tampuan",
+    "vie": "Vietnamese",
+    "war": "Waray-Waray",
+    "wms": "Wambon",
+    "wnk": "Wanukaka",
+    "xmm": "Manado Malay",
+    "yet": "Yetfa",
+    "zlm": "Malay",
+}
+
+#it's a gated dataset, hence _LOCAL = True
+_LOCAL = True
+_LANGUAGES = list(_LANG_CONFIG.keys())
+
+
+_DATASETNAME = "bloom_captioning"
+_DESCRIPTION = r"""
+    This is a Bloom Library dataset developed for the image captioning task.
+    It covers 74 languages indigenous to SEA overall, amounting to total data of 21K.
+    This dataset belongs to a CC license, where its datapoints has specific license attached to it.
```

Comment:
Tried and tested. The dataloader works well. One minor thing, though:

Since the HuggingFace URL is only accessible if the user has accepted the dataset's acknowledgement, could we append `"Before using this dataloader, please accept the acknowledgement at https://huggingface.co/datasets/sil-ai/bloom-captioning and use huggingface-cli login for authentication."` to the `_DESCRIPTION`?